spec/support/factories.cr

```diff
@@ -34,6 +37,11 @@ end
 def create_fork_git_repository(project, upstream)
   Dir.cd(tmp_path) do
     run "git clone #{Process.quote(git_url(upstream))} #{Process.quote(project)}"
+
+    Dir.cd(Process.quote(project)) do
```

Comment:
`Process.quote` is invalid here. Oh oops, there's actually another invalid instance of it already checked in. Maybe let's just remove both instances here?
```suggestion
    Dir.cd(project) do
```

Reply:
Thank you @oprypin for pointing it out, looking a bit deeper into the code found that it should be `Dir.cd(git_path(project))` instead, to be consistent with the other usage across the test codebase.

Cheers.
❤️ ❤️ ❤️

Reply:
It's actually not necessary because the `git_path` part is already covered by `Dir.cd(tmp_path)` above, but fine either way.

Reply:
Yeah, thought that, but to be consistent on how is being used in several places: 

https://github.com/crystal-lang/shards/blob/master/spec/support/factories.cr#L40-L54

Thought will be good to be consistent 😊